src/Proto.Actor/LocalContext.cs

```diff
@@ -286,25 +289,25 @@ public void EscalateFailure(PID who, Exception reason)
 
         public void RestartChildren(params PID[] pids)
         {
-            for (int i = 0; i < pids.Length; i++)
+            foreach (var pid in pids)

```

Comment:
Cosmetic reasons only, this is a non-critical path in terms of performance.
it will occur occasionally only.